mnist/mnist-webapp/README.md

```diff
@@ -0,0 +1,23 @@
+# Mnist Digit Reader
```

Comment:
Why do we need to copy the mnist-webapp code to the example repo? Is there a docker container already that we can use?

Reply:
there is no docker container, but your suggestion is good. I can build a minist-webapp container and then remove this mnist-webapp code.

Thank you very much.

Reply:
Thanks.